torch/nn/parallel/distributed.py

```diff
@@ -316,6 +316,15 @@ class DistributedDataParallel(Module):
                          are getting different gradients, which should not
                          happen if DistributedDataParallel is correctly used.
                          (default: ``False``)
+        grad_is_view: when setting to ``True``, grads will be views of all reduce
+                      buckets in reducer, this can reduce peak memory usage, the
+                      saved memory size will be equal to the total grads size.
+                      Also it can avoid coping between grads and the all reduce
+                      buckets in reducer and thus improve training performance.
+                      When grads are views, detach_() can not be called in your
```

Comment:
grads -> gradients

double quote `detach_()`

> can not be called in your zero_grad() function, please

cannot be called on the gradients.  If hitting such errors, please